__tests__/topics.test.js

```diff
@@ -19,6 +19,57 @@ describe('/api/topics', () => {
   });
 });
 
+describe('/api/articles', () => {
+  describe('GET /api/articles/:article_id', () => {
+    test('200: status code and contain the expected data type and fields', async () => {
+      const res = await request(app).get(`/api/articles/33`);
+      expect(res.statusCode).toBe(200);
+      expect(Array.isArray(res.body)).toBe(true);
+      res.body.forEach(article => {
+        expect(article).toEqual(
+          expect.objectContaining({
+            article_id: expect.any(Number),
```

Comment:
I would be specific here and check that the article_id is 33 - proving you have the correct one. You can be specific with the other properties too but at a minimum check this